### ISSUE TITLE
Pip Installation faraday-telemetry Not Creating telemetrydb.sql In etc\faraday Folder

### ISSUE THREAD

**Issue by kb1lqd** [FaradayRF/Faraday-Software#216]
## Summary
I started from a fresh `...\etc\faraday` folder and pip installed faraday. I found that proxy ran successfully but when trying to initialize and run faraday-telemetry the program could not find the needed SQL file and crashed.

## Problem Explanation
I've pasted an image of the error below in supporting information. 

I was able to successfully run telemetry after manually copying the `db.sql` file into the `..\etc\faraday\` folder and renaming it to `telemetrydb.sql` from the master github repository.

#### Software 
Pip installed faraday version information.
```
C:\Users\Brent>pip show faraday
Name: faraday
Version: 0.0.1013.dev1040
Summary: FaradayRF amateur radio open source software
Home-page: https://github.com/FaradayRF/Faraday-Software
Author: FaradayRF
Author-email: Support@FaradayRF.com
License: GPLv3
Location: c:\python27\lib\site-packages
Requires: pytest-flake8, six, py, requests, Flask-Cors, packaging, pyflakes, MarkupSafe, itsdangerous, appdirs, pyserial, pycodestyle, Werkzeug, Flask, pyparsing, Jinja2, click, flake8, pytest, configparser, mccabe, enum34
```
Master github software.

#### Hardware
Faraday REV D1 units.

# Supporting Information

Error received:
![image](https://user-images.githubusercontent.com/339144/27646769-60c577ee-5bde-11e7-8ff3-47799937eb79.png)

Github repo telemetry db.sql file, I renamed this to telemetrydb.sql and moved into the etc/faraday folder.

![image](https://user-images.githubusercontent.com/339144/27646799-78ceeffa-5bde-11e7-86f3-1a5fc103c33d.png)

The etc/faraday folder. Note that the `db.sql` file is for proxy not telemetry.

![image](https://user-images.githubusercontent.com/339144/27646821-89f11290-5bde-11e7-9ad9-ab946248dbd1.png)

Succesfully running telemetry after manually copying over telemetrydb.sql file.

![image](https://user-images.githubusercontent.com/339144/27646866-a71dd01a-5bde-11e7-97e2-03acd7187ab4.png)

**Comment by kb1lqc** [comment]
Installing with `pip install --no-cache-dir faraday` results in this working for me just fine. However I just realized that's because I haven't cleared my `telemetry.db` which will persist between uninstalls apparently.

![image](https://user-images.githubusercontent.com/331540/27672757-d05efde6-5c51-11e7-804b-c4e7742b4168.png)

This results in the new installation not having to create a database and therefore not needing the schema. Deleting `telemetry.db` results in the error since I now run the code that creates a database from the `telemetrydb.sql` file:

![image](https://user-images.githubusercontent.com/331540/27672743-ba3021a8-5c51-11e7-8a8e-1c8ff154de79.png)

So confirmed...

# The Fix
Simply adding `etc/faraday/telemetrydb.sql` to `setup.cfg` is all that's needed. This tells setuptools to copy the local `telemetrydb.sql` into the main `/etc/faraday` folder. Testing this with a locally created wheel distribution file I successfully installed faraday and obtained the needed file.

![image](https://user-images.githubusercontent.com/331540/27673043-70e2178e-5c53-11e7-9ba9-59a4a9a4b7a8.png)

Starting with a clean `.faraday/` folder:

![image](https://user-images.githubusercontent.com/331540/27673069-96ba66dc-5c53-11e7-9e95-ee66b247bdd2.png)

Launching `faraday-telemetry` worked fine:
![image](https://user-images.githubusercontent.com/331540/27673088-af9b4a18-5c53-11e7-8143-afcba87b2e6c.png)

You can see that the `telemetry.db` log file was created successfully!
![image](https://user-images.githubusercontent.com/331540/27673106-c8f8bd1a-5c53-11e7-9348-1f552f3c29e3.png)

Fixed in PR#217

**Comment by kb1lqd** [comment]
#217 closes this.
